plugins/versions.properties

```diff
@@ -28,3 +28,5 @@ version.kotlinx.coroutines=1.3.8
 version.okhttp3=4.8.1
 
 version.retrofit2=2.9.0
+
+version.moshi=1.10.0
```

Comment:
I'm downgrading to version 1.9.3 because the version 1.10.0 depends on Kotlin stdlib 1.4.0, which is not supported in Gradle 6.6 and might cause subtle issues.

We'll upgrade again after Gradle 6.7 which should have Kotlin 1.4 compatibility.